### ISSUE TITLE
Track game session information

### ISSUE THREAD

**Issue by JosefNemec** [JosefNemec/Playnite#1992]
Playnite currently doesn't save information about separate game sessions, it only adds session play time to the overall game play time. This is an issue if we want to show more advanced statistics, like play time for past week.

**Comment by gagrecco** [comment]
It would be cool to have a whole play time record like on the Wii. At the end pf the day you can check how much you play each game and everything was recorded like "Wii sports - 2:34hr, TP - 20 min, total play time - 2:54" etc. And it got save

**Comment by RalfRoemling** [comment]
+1

**Comment by chocolatechipcats** [comment]
If this is still on the table, here are some options I would like:

- Option to import already-existing GameActivity logs;
- Option to manually add, edit, and merge sessions;
- Option to copy/move sessions from one game to another (example: "upgrading" a DRM'd disc copy to the GOG version)

**Comment by chocolatechipcats** [comment]
Yes -- currently, GameActivity just ignores the pre-existing time. But assuming it can be done, perhaps a "time unaccounted for" value that subtracts the known sessions from the existing playtime may work?

**Comment by chocolatechipcats** [comment]
> If this is still on the table, here are some options I would like:
> 
> * Option to import already-existing GameActivity logs;
> * Option to manually add, edit, and merge sessions;
> * Option to copy/move sessions from one game to another (example: "upgrading" a DRM'd disc copy to the GOG version)

Also, fourth:  Which play action is being used. Expansions, for example.
